tests/integration/utils.sh

```diff
@@ -51,6 +82,10 @@ function setup_container_environment()
   local distros_ok # array of distros whose setup succeeded
   local distro
 
+  if [[ "$verbose" -eq 1 ]]; then
+    say "Preparing environment for kw ${feature} integration tests..."
```

Comment:
Could you please clarify why you're adding this? I've already included a [commit](https://github.com/kworkflow/kworkflow/commit/b6420c5ae783531c09844fbafd53d0a4514a2cf8#diff-60f0b4fc0c305f055f1f368deef91ba9eb634dff1a5569e058e78108eb82db1bR57) that accomplishes the same task